app/routes/password/reset.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+const { Route, get, inject: { service } } = Ember;
+
+export default Route.extend({
+
+  ajax: service(),
+  session: service(),
+
```

Comment:
One thing I need here is a model hook that extracts the token from `params` and used to send up with the POST request.